### PR TITLE
support RHEL and Rocky 9

### LINKED ISSUE #243: Rocky 9 is out
uses CodeReadyBuilder like RHEL9.
update role to support this platform once Vagrant/Docker/other container images exist.

### PR DESCRIPTION
**Pull request by donsizemore**
Closes #243

### CODE REVIEW

File: tasks/dataverse-build.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@
     create: yes
 
 - name: build warfile without tests and default maven. tail /tmp/dataverse/mvn.out for gory details.
-  shell: "export JAVA_HOME={{ java.home }} && mvn -T 2C clean package -Dmaven.test.skip=true > mvn.out"
+  shell: "export JAVA_HOME={{ java.version }} && mvn -T 2C clean package -Dmaven.test.skip=true > mvn.out"
```
Comment (pdurbin) [inline tasks/dataverse-build.yml]: Is this right, this change from java version to home?

**Review by pdurbin — COMMENTED** [review]
Thanks, @donsizemore!

Overall, looks ok. I'm glad this is in there, as discussed:

```
# let's stick with rocky 8.9 until ITs pass
AWS_AMI_DEFAULT='ami-0408f4c4a072e3fb9'
```

I left a comment about a java home/version thing.

Also, there seem to be additional changes that might be off topic (stuff about localstack/minio) but probably I'm just not close enough to it.

File: tasks/dataverse-build.yml (hunk 1 above)
Comment (donsizemore) [reply tasks/dataverse-build.yml]: @pdurbin that is not right, and gives me pause. I wasn't sure why I saw all the MinIO/LocalStack stuff, like it wanted to replay old commits. I'll compare them against current develop.

**Comment by donsizemore** [comment]
@pdurbin I cleaned up those commits - one more review?

File: tasks/dataverse-prereqs.yml [hunk 2]
```diff
@@ -46,16 +46,23 @@
     name: ['bash-completion', 'git', 'jq', 'mlocate', 'net-tools', 'sudo', 'unzip', 'python3-psycopg2', 'zip', 'tar']
     state: latest
 
-- name: "RHEL/Rocky 8.6 provides Ansible 5.4 which wants Python-3.8"
+- name: "RHEL/Rocky 8.6 packaged Ansible wants Python-3.8"
```
Comment (pdurbin) [inline tasks/dataverse-prereqs.yml]:
```suggestion
- name: "RHEL/Rocky 8.6-packaged Ansible wants Python-3.8"
```

... or something to make this a bit more clear. The old "provides" and "which" actually reads better to me.

What I think we're saying:

The version of Ansible provided by RHEL/Rocky 8.6 requires Python 3.8.

I'm not sure if I have this right though. 😅

File: tasks/dataverse-prereqs.yml [hunk 3]
```diff
@@ -46,16 +46,23 @@
...
   ansible.builtin.package:
     name: ['python38-psycopg2']
     state: latest
   when: ansible_os_family == "RedHat" and
         ansible_distribution_major_version == "8"
 
+- name: "RHEL/Rocky 9 provides 3.9"
```
Comment (pdurbin) [inline tasks/dataverse-prereqs.yml]:
```suggestion
- name: "RHEL/Rocky 9 provides Python 3.9"
```

**Review by pdurbin — COMMENTED** [review]
Looks great, just a couple wording suggestions.

**Pull request merged**